man/systemd.exec.xml

```diff
@@ -1321,6 +1321,19 @@ BindReadOnlyPaths=/var/lib/systemd</programlisting>
         <xi:include href="system-only.xml" xpointer="singular"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ProtectKernelLogs=</varname></term>
+
+        <listitem><para>Takes a boolean argument. If true, access to the kernel ring buffer will be denied. It is
+        recommended to turn this on for most services that do not need to read kernel logs. Enabling this option
+        removes <constant>CAP_SYSLOG</constant> from the capability bounding set for this unit, and installs a system
+        call filter to block the <citerefentry project='man-pages'><refentrytitle>syslog</refentrytitle><manvolnum>2</manvolnum></citerefentry>
+        system call. The kernel exposes logs to userspace via <filename>/dev/kmsg</filename> and
```

Comment:
"system call" → "system call (not to be confused by the libc API syslog(3) for userspace logging)"